src/RGUnityBots/Assets/Scripts/LegacyInputTests/LegacyPointerHandlerUniRx.cs

```diff
@@ -37,26 +36,27 @@ static T GetOrAddComponent<T>(GameObject gameObject)
         }
     }
 
-    public class LegacyPointerHandler: MonoBehaviour
+    public class LegacyPointerHandlerUniRx: MonoBehaviour
     {
-        void Update()
-        {
-            // if you call this like unity documents during startup, it gets removed again.. so dumb
-            TouchSimulation.Enable();
-        }
+        public GameObject clickedIndicator;
 
         public void Start()
         {
 
             gameObject.OnPointerDownAsObservable()
-                .SelectMany(_ => this.gameObject.UpdateAsObservable())
-                .TakeUntil(this.gameObject.OnPointerUpAsObservable())
-                .Select(_ => Input.mousePosition)
-                .RepeatUntilDestroy(this) // safety way
                 .Subscribe(
                     x =>
                     {
                         Debug.Log($"{gameObject.name} OnPointerDownAsObservable()");
+                        clickedIndicator.SetActive(true);
```

Comment:
changes this to be verbosely on the down and up separately like the customer sample